Aquib/1_Valid_Sudoku.c.txt

```diff
@@ -0,0 +1,83 @@
+bool checkRow(char **board, int n)
```

Comment:
Also, let's add the leetcode question link in the description

Reply:
Yesss. I have been adding the link to the readme file. We can also add it in the file that we commit as well.

Reply:
perfect

Reply:
Let's add the time and space complexity of the solution in the description.

Reply:
Great Idea. I will update all the changes and recommit.

Reply:
Don't forget to add time and space complexity in the description